src/Log/Engine/ConsoleLog.php

```diff
@@ -57,14 +57,6 @@ class ConsoleLog extends BaseLog
      */
     public function __construct(array $config = [])
     {
-        if ((DIRECTORY_SEPARATOR === '\\' && !(bool)env('ANSICON') && env('ConEmuANSI') !== 'ON') ||
-            (function_exists('posix_isatty') && !posix_isatty($this->_output))
```

Comment:
`$this->_output` property is unset at this point so this check makes no sense.

Reply:
Does changing the output style to plain exist somewhere else too? Without this we will end up outputting bash escape codes into redirected output handles.

Reply:
Yes, ConsoleOutput's constructor does similar check in a proper manner and switches to plain text output if needed